### ISSUE TITLE
Visual Studio style hotkeys

### ISSUE THREAD

**Issue by JVimes** [lukebuehler/CShell#23]
I keep wanting to hit F5 to run instead of Alt+Shift+Return. I'd like to see the hotkeys mimic those of Visual Studio. I suppose the ultimate solution is configurable hotkeys, with switchable schemes and one of them being Visual Studio style.

**Comment by lukebuehler** [comment]
Yes, this is a good idea, will add it as a planned feature. Some kind of key-mappings configuration would be good for the editor, I think AvalonEdit already has partial support for this.
